pom.xml

```diff
@@ -102,10 +102,10 @@ under the License.
   </properties>
 
   <scm>
-    <url>https://git-wip-us.apache.org/repos/asf?p=crunch.git</url>
-    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</connection>
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</developerConnection>
-    <tag>apache-crunch-0.13.0</tag>
+    <url>https://github.com/scalingdata/crunch</url>
+    <connection>scm:git:git@github.com:scalingdata/crunch.git</connection>
+    <developerConnection>scm:git:git@github.com:scalingdata/crunch.git</developerConnection>
+    <tag>0.13.0-hdp2.3.0-rocana</tag>
```

Comment:
Oh wait, just re-read your above comment:
> Tested with CRUNCH_RELEASE_VERSION=`0.13.0-hdp2.3.0-rocana1.3.0-rc4`

I see, maven's `<tag>` is a branch name.
Nevermind my comment above, carry on. Nothing to see here.